diff --git a/couchpotato/core/downloaders/nzbget.py b/couchpotato/core/downloaders/nzbget.py
index b46de77..b8626af 100644
--- a/couchpotato/core/downloaders/nzbget.py
+++ b/couchpotato/core/downloaders/nzbget.py
@@ -239,8 +239,8 @@ config = [{
                 },
                 {
                     'name': 'host',
-                    'default': 'localhost:6789',
-                    'description': 'Hostname with port. Usually <strong>localhost:6789</strong>',
+                    'default': 'localhost:9092',
+                    'description': 'Hostname with port. Usually <strong>localhost:9092</strong>',
                 },
                 {
                     'name': 'ssl',
@@ -251,14 +251,13 @@ config = [{
                 },
                 {
                     'name': 'username',
-                    'default': 'nzbget',
-                    'advanced': True,
-                    'description': 'Set a different username to connect. Default: nzbget',
+                    'default': 'xbian',
+                    'description': 'Set a different username to connect. Default: xbian',
                 },
                 {
                     'name': 'password',
                     'type': 'password',
-                    'description': 'Default NZBGet password is <i>tegbzn6789</i>',
+                    'description': 'Default NZBGet password is <i>raspberry</i>',
                 },
                 {
                     'name': 'category',
